eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -49,8 +50,6 @@ public function __construct(UrlRedecoratorInterface $redecorator, Connection $co
     /**
      * Return the node path string of $versionInfo.
      *
-     * @param \eZ\Publish\SPI\Persistence\Content\VersionInfo $versionInfo
-     *
      * @return string
```

Comment:
Why remove this?

Reply:
> Why remove this?

Because this information is already available in the method signature.